### DISCUSSION TITLE
匿名命令问题

### DISCUSSION THREAD

**Discussion by pkeke** [note286/xduts#94] [Q&A] [answered]
导师的匿名功能失效
![image](https://user-images.githubusercontent.com/44956389/227156831-20d01fc3-9a1d-4bba-a132-880f7a74c00f.png)
代码如下：
```
        supervisor = {\anon[XXX]{123}},
        supervisor* = {\anon[XXX]{123}},
        supervisor-enterprise = {\anon[XXX]{456}},
        supervisor-enterprise* = {\anon[XXX]{456}},
```

**Comment by note286** [comment, marked as answer]
查看 https://github.com/note286/xduts/issues/61 中的评论。

**Reply by pkeke** [reply]
好的，谢谢！

**Reply by Nono-404** [reply]
> 查看 #61 中的评论。

![image](https://user-images.githubusercontent.com/56815855/228181710-88c79bdf-10e3-455a-a554-55ffb9c48232.png)
在个人简介里使用\anno{}报错：Undefined control sequence.大佬知道这是什么问题吗

**Reply by note286** [reply]
请更新最新版xduts。

**Reply by note286** [reply]
你的问题和当前主题无关，请新开一个。
